include/tables/hive/SCHEMAVERSIONTable.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of ePipe
+ * Copyright (C) 2020, Logical Clocks AB. All rights reserved
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+
+#ifndef EPIPE_SCHEMAVERSIONTABLE_H
+#define EPIPE_SCHEMAVERSIONTABLE_H
+
+#include "tables/DBTable.h"
+
+struct SCHEMAVERSIONRow {
+  Int64 mSCHEMAID;
+  Int64 mSERDEID;
+  Int64 mCDID;
+};
+
+class SCHEMAVERSIONTable : public DBTable<SCHEMAVERSIONRow> {
+
+public:
+  SCHEMAVERSIONTable() : DBTable("SCHEMA_VERSION") {
+    addColumn("SCHEMA_ID");
+    addColumn("SERDE_ID");
+    addColumn("CD_ID");
+  }
+
+  SCHEMAVERSIONRow getRow(NdbRecAttr *value[]) {
+    SCHEMAVERSIONRow row;
+    row.mSCHEMAID = value[0]->int64_value();
+    row.mSERDEID = value[1]->int64_value();
+    row.mCDID = value[2]->int64_value();
+    return row;
+  }
+
+  void removeBySchemaID(Ndb *conn, Int64 SCHEMAID)
+  {
+    AnyMap key;
+    key[0] = SCHEMAID;
+    int count = deletesByIndex(conn, "SIIndex", key, boost::none);
```

Comment:
Can we add a partition key to these tables?

Reply:
same as above